Revamp README (#53)
scientific-python/executable-tutorials: README.md

# Executable Tutorials

## Use Cases

Many organizations maintain collections of tutorials addressing realistic
problems in a science domain, with runnable code examples written and
kept current by experts in the tools. They are useful as:

- A trove of working snippets to copy and paste
- A maintained resource for self-guided learning

> - A ready-to-use curriculum for interactive workshops

Reviewer: I'm not sure what interactive workshop mean
```suggestion
- A ready-to-use curriculum for hands-on workshops
```

> - A suite of science domain-specific "integration tests"

Reviewer: ```suggestion
- A suite of science domain-specific "integration tests" for the showcased libraries
```

## Key Features

This repository demonstrates one way to configure a collection of tutorials,
covering some core features:

> - Source is in [MyST Markdown][] which is easy for humans to edit and review.

Reviewer: Maybe link to this?
```suggestion
- Source is in [MyST Markdown](https://mystmd.org/guide/md-vs-ipynb) which is easy for humans to edit and review.
```

Reviewer: haha, my bad of not knowing the [link][] syntax...)

# Executable Tutorials

## Use Cases

Many organizations maintain collections of tutorials addressing realistic
problems in a science domain, with runnable code examples written and
kept current by experts in the tools. They are useful as:

- A trove of working snippets to copy and paste
- A maintained resource for self-guided learning
- A ready-to-use curriculum for interactive workshops
- A suite of science domain-specific "integration tests"

## Key Features

This repository demonstrates one way to configure a collection of tutorials,
covering some core features:

- Source is in [MyST Markdown][] which is easy for humans to edit and review.
- The executed examples---code and results---are published as a static site.

>   ([example][static site example])

Reviewer: Having a bit of "click here" vibe from the example links, maybe work them into the text here and below?

https://heyoka.medium.com/dont-use-click-here-f32f445d1021
